api/Trace/Span.php

```diff
@@ -58,6 +58,14 @@ public function addEvent(string $name, int $timestamp, ?Attributes $attributes =
      */
     public function addLink(SpanContext $context, ?Attributes $attributes = null): Span;
 
+    /**
+     * @param string $name
+     * @param int $timestamp
+     * @param Attributes|null $attributes
+     * @return Span Must return $this to allow setting multiple attributes at once in a chain.
+     */
+    public function recordException(string $name, int $timestamp, ?Attributes $attributes = null): Span;
```

Comment:
According to [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#record-exception) the Span should accept `Exception` object (`Throwable` interface) instead of just its' name. The name of the event [will be](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/exceptions.md#recording-an-exception) `"exception"`.